quinn-proto/src/connection.rs

```diff
@@ -2726,18 +2756,22 @@ impl Connection {
         self.key_phase = !self.key_phase;
     }
 
+    /// If the connection is currently handshaking
     pub fn is_handshaking(&self) -> bool {
         self.state.is_handshake()
     }
 
+    /// If the connection is closed
     pub fn is_closed(&self) -> bool {
         self.state.is_closed()
     }
 
+    /// If the connection is drained
     pub fn is_drained(&self) -> bool {
         self.state.is_drained()
     }
 
+    /// If the connection accepted 0-RTT streams
```

Comment:
```suggestion
    /// If the connection accepted 0-RTT streams after becoming established
```

Reply:
I don't quite understand what you mean here, can you elaborate/rephrase?

Reply:
This value may only become true after the connection is established. I'm not sure we should guarantee any particular value prior to that, but at the moment it always starts out false.

Reply:
Tried to improve this, have a look.